src/browser/navigation-panel.js

```diff
@@ -100,12 +101,15 @@ define((require, exports, module) => {
     return DOM.div({
       key,
       style: theme.navigationPanel,
-      className: 'navbar' + (inputCursor.get('isFocused') ? ' urledit' : '') +
-                            (webViewerCursor.get('canGoBack') ? ' cangoback' : '') +
-                            (webViewerCursor.get('location') ? ' canreload' : '') +
-                            (webViewerCursor.get('isLoading') ? ' loading' : '') +
-                            (webViewerCursor.get('securityState') == 'secure' ? ' ssl' : '') +
-                            (webViewerCursor.get('securityExtendedValidation') ? ' sslv' : '')
+      className: ClassSet({
+        navbar: true,
+        urledit: inputCursor.get('isFocused'),
+        cangoback: webViewerCursor.get('canGoBack'),
+        canreload: webViewerCursor.get('location'),
+        loading: webViewerCursor.get('isLoading'),
+        ssl: webViewerCursor.get('securityState'),
```

Comment:
We used to compare `webViewerCursor.get('securityState') == `secure`. What is the contents of `webViewerCursor.get('securityState')`? `secure` or `null`? Want to make sure we're not introducing a state bug.